src/main/java/com/gregtechceu/gtceu/data/lang/MaterialLangGenerator.java

```diff
@@ -2,15 +2,19 @@
 
 import com.gregtechceu.gtceu.api.data.chemical.material.Material;
 import com.gregtechceu.gtceu.api.data.chemical.material.registry.MaterialRegistry;
+import com.gregtechceu.gtceu.common.data.GTMaterials;
 import com.tterrag.registrate.providers.RegistrateLangProvider;
 
 import static com.gregtechceu.gtceu.utils.FormattingUtil.toEnglishName;
+import static com.gregtechceu.gtceu.data.lang.LangHandler.replace;
 
 public class MaterialLangGenerator {
 
     public static void generate(RegistrateLangProvider provider, MaterialRegistry registry) {
         for (Material material : registry.getAllMaterials()) {
             provider.add(material.getUnlocalizedName(), toEnglishName(material.getName()));
         }
+
+        replace(provider, GTMaterials.RTMAlloy.getUnlocalizedName(),"RTM Alloy");
```

Comment:
Wrong class

Reply:
What class should I put this in?

Reply:
LangHandler, with the HSS-G replacement etc.

Reply:
because MaterialLangGenerator generates per-MOD, including addons